src/dram_system.cc

```diff
@@ -163,9 +182,9 @@ void JedecDRAMSystem::ClockTick() {
     }
     clk_++;
 
-    if (clk_ % config_.epoch_period == 0) {
-        PrintEpochStats();
-    }
+//    if (clk_ % config_.epoch_period == 0) {
```

Comment:
Do we need to break the epoch_period functionality? Maybe PrintTagStats rather than PrintEpochStats since it's somewhat orthogonal?

Reply:
Let's keep periodic printing, but emit the tag stats to a different file.

@marcusnchow is that doable?

Reply:
Maintain old functionality, append new functionality
(It will mean two .json files, but the alternative is turning off the old file -- which is fine too)

Reply:
i turned it off because there are two statsprint epoch period, one from hammerblade and one from dramsim, so with this dramsim prints when hammerblade prints

Reply:
ye that should work as well

Reply:
Maybe I'm not following. The manycore stats are printed when the profiler calls are made, and your work added that to dramsim3.

Reply:
omg, no im just loosing it, i understand what your saying now, i know how i can do that know